Lib/inspect.py

```diff
@@ -2805,7 +2805,8 @@ def __init__(self, parameters=None, *, return_annotation=_empty,
 
                     params[name] = param
             else:
-                params = {param.name: param for param in parameters}
+                params = OrderedDict(((param.name, param)
```

Comment:
Extra parens are not needed. This could be `params = OrderedDict((param.name, param) for param in parameters)`

Reply:
I mostly copied this code from the 3.8 source, weird that https://github.com/python/cpython/blob/3.8/Lib/inspect.py#L2809 has this. I'll remove the parens and the enumeration above.